03_letters/CameronSchafer/letters_test.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// main() test
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+	expected := "a:2\nb:1\n"
+	actual := buf.String()
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main()\nexpected: %q\nactual: %q", expected, actual)
+	}
+}
+
+// sortLetters()
+func TestSortLetters(t *testing.T) {
+	// test cases with descriptions.
+	testCases := []struct {
+		description string
+		input       map[rune]int
+		expectedArr []string
+	}{
+		{description: "sortLetters [97:2, 98:1]", input: map[rune]int{97: 2, 98: 1},
```

Comment:
Isn't going to be easier to understand the tests if you use a rune literal, instead of the integer representation.

e.g. replace, 97 with 'a', etc, etc.

Reply:
I updated the test cases to the rune literals. I didn't even know I could do that! 👍